packages/utils/package.json

```diff
@@ -4,6 +4,7 @@
     "version": "1.0.0",
     "main": "src/index.ts",
     "dependencies": {
+        "bowser": "^2.9.0",
```

Comment:
why we need this here?

Reply:
This dependency is used in `getLoginHistoryFormattedData()` of utils package inside Deriv app.